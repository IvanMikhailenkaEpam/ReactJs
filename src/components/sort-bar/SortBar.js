import React from 'react';
import * as PropTypes from 'prop-types';

const SortBar = ({ children }) => (
  <div className="pt-3 pb-3 gray-back">
    <div className="container">
      <div className="row mr-3 pr-3">
        {children}
      </div>
    </div>
  </div>
);

SortBar.propTypes = {
  children: PropTypes.element.isRequired,
};

export default SortBar;
