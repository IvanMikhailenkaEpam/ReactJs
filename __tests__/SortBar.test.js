import React from 'react';
import { shallow } from 'enzyme';
import SortBar from "../src/components/sort-bar/SortBar";

describe('SortBar', () => {
  it('renders SortBar correctly', () => {
    const component = shallow(<SortBar />);

    expect(component).toMatchSnapshot();
  });
});
