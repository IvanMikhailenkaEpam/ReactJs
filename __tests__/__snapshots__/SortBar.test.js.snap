// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortBar renders SortBar correctly 1`] = `
<div
  className="pt-3 pb-3 gray-back"
>
  <div
    className="container"
  >
    <div
      className="row mr-3 pr-3"
    />
  </div>
</div>
`;
